### PR TITLE
Fix #878 - Adds option to view client image

### LINKED ISSUE #878: Feature: Adding option to view client image in client section.
**Summary:** 

Adding option to view client image in client section currently, there is no such option to view the image on full screen.

**Steps to reproduce:** 
Clients->Pick a Client->Click on image.

**Screenshots:** 
screenshot_20180211-041559

### PR DESCRIPTION
**Pull request by OussEmaDevCode**
Fixes #878 

Please Add Screenshots If there are any UI changes.
video2gif_20191208_194101
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by santoshconflux** [comment]
@elirehema, @moksh-mahajan , please review.

**Comment by moksh-mahajan** [comment]
The changes look fine to me. @luckyman20 @elirehema Can you take a quick look?

**Comment by elirehema** [comment]
Am working on it @moksh-mahajan

**Comment by elirehema** [comment]
Good job @OussEmaDevCode

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/ViewImage.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.mifos.mifosxdroid;
```
Comment (luckyman20) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/ViewImage.java]: Change activity name to `ViewImageActivity`

File: mifosng-android/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -153,6 +153,10 @@
         <service android:name=".activity.pathtracking.PathTrackingService"
             android:exported="false"/>
 
+        <activity
+            android:name=".ViewImage"
+            android:label="@string/view_image"
```
Comment (luckyman20) [inline mifosng-android/src/main/AndroidManifest.xml]: Either use `View Image` or `View Photo`, not both.

**Comment by OussEmaDevCode** [comment]
@luckyman20 Done !!

File: mifosng-android/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -791,5 +792,4 @@
     <string name="status_head">status</string>
     <string name="client_search_hint"></string>
 
-
```
Comment (laxyapahuja) [inline mifosng-android/src/main/res/values/strings.xml]: Unnecessary.

**Comment by laxyapahuja** [comment]
@OussEmaDevCode your commit message description includes unnecessary messages. i think it'll be better to remove them or add an actual description. :)

**Comment by OussEmaDevCode** [comment]
@laxyapahuja That line is a default addition in the file and as the contribution guideline specifies, you're only supposed to change parts that are related to your issue.

**Comment by OussEmaDevCode** [comment]
And for description, these were added automatically after debasing, but I'll obviously remove them after fixing the merge conflicts.

**Comment by Aditya-gupta99** [comment]
We are closing this PR as it causes a branch conflict since the Java file has now been migrated to kotlin

**Pull request closed without merging**